omnigan/trainer.py

```diff
@@ -1622,9 +1633,15 @@ def resume(self, inference=False):
         # -----  Single Model Loading  -----
         # ----------------------------------
         else:
-            load_path = Path(self.opts.output_path) / Path(
-                "checkpoints/latest_ckpt.pth"
-            )
+            if self.opts.train.resume_by_pth_path:
+                load_path = Path(self.opts.output_path)
+                assert (
+                    os.path.splitext(load_path)[-1] == ".pth"
+                ), "The path is not a pth checkpoint file."
+            else:
+                load_path = Path(self.opts.output_path) / Path(
```

Comment:
no need to have a Path after `/`. `Path() / "string" / "some-other-string"` works fine